internal/cache/avi_ctrl_clients.go

```diff
@@ -52,6 +52,9 @@ func SharedAVIClients(tenant string) *utils.AviRestClientPool {
 			lib.AKOShutdown, "Avi Controller information missing (username: %s, password: %s, authToken: %s, controller: %s)",
 			ctrlUsername, passwordLog, authTokenLog, ctrlIpAddress,
 		)
+		if ctrlIpAddress == "" {
+			utils.AviLog.Fatalf("Avi Controller information missing (username: %s, password: %s, authToken: %s, controller: %s). Update the controller IP in installation configuration", ctrlUsername, passwordLog, authTokenLog, ctrlIpAddress)
```

Comment:
should we mention configmap here instead of installation configuration?

Reply:
updated with configmap